dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -188,6 +191,30 @@ public UserResource(HttpClient client, SessionFactory sessionFactory, WorkflowRe
         this.bitbucketClientSecret = configuration.getBitbucketClientSecret();
     }
 
+    /**
+     * Checks if the include string (csv) includes some field
+     * @param include CSV string
+     * @param field Field to query for
+     * @return True if include has the given field, false otherwise
+     */
+    private boolean checkIncludes(String include, String field) {
+        String includeString = (include == null ? "" : include);
+        ArrayList<String> includeSplit = new ArrayList(Arrays.asList(includeString.split(",")));
```

Comment:
I don't think you need an `ArrayList` wrapping the list. Just `List<String> includeSplit = Arrays.asList(includeString.split(","))`

Later...

Ah I see, you copied this from existing code.

Bonus: would be nice to eliminate the code duplication, but I can see how that would be a hassle, because the existing code is an interface that you don't implement. So probably don't bother.

Reply:
Good call on the array list, fixed: https://github.com/dockstore/dockstore/pull/4837/commits/ea0b684150b2efe00300ec862f9bd951d95b0dc6

Yeah, I think the only function needed from the interface is `checkIncludes`, so some minor duplication for now might be OK.